dept44-formatting-plugin/pom.xml

```diff
@@ -11,8 +11,6 @@
 	<packaging>maven-plugin</packaging>
 	<properties>
 		<maven-api.version>3.9.9</maven-api.version>
-		<maven-plugin-annotations.version>3.15.1</maven-plugin-annotations.version>
```

Comment:
This is not part of you PR, but still:

- maven-plugin-annotations.version is also defined in build-tools-module. Could be removed there as well?
- maven-api.version is used in build-tools-module (hence defined in thwo places in dept44) Move version definition to parent-pom?